### ISSUE TITLE
How to find the optimal hyper parameter introduced by the author?

### ISSUE THREAD

**Issue by sisrfeng** [xingyizhou/CenterNet#812]
![image](https://user-images.githubusercontent.com/53520949/92456592-b427a500-f1f5-11ea-9eec-3021f9fecd78.png)

> We set λsize = 0.1 and λoff = 1 in all our experiments unless specified otherwise.
> Table 3b shows 0.1 gives a good result. For larger values, the AP degrades significantly, due to the scale of the loss ranging from 0 to output size w/R or h/R, instead of 0 to 1.

1. What about other values like 0.8,0.9 (<1)? Why should the loss range from 0 to 1?
2. Is  λoff not so relevant to the final result? 

3. random scaling (between 0.6 to1.3) . What about larger or smaller?
4. multi-scale ---0.5, 0.75, 1, 1.25, 1.5. What about 0.25 and 1.75? Using 0.25 can only detect quite large object and using 1.75 is too slow?

5. input size: 512x512,384x384,  why not 256x256 ? (half of 512)

 6. On VOC: We do not use Hourglass-104 [30] because it fails to converge in a reasonable time (2 days) when trained from scratch. 
Why not finetune?
> we fine-tune the Hourglass-104 from ExtremeNet  to save computation.

Many thx!

**Comment by xingyizhou** [comment]
Thank you for your questions.
1. I usually tune hyper-parameters in a log-scale like 1/ 0.5/ 0.25/ 0.1. 
2. It is not as far as I remember.
3/4. These are from CornerNet, I haven't played with them.
5. 256 would be much worse than 384 as far as I remember.
6. Finetuning VOC from COCO is usually much better (>2mAP@0.5) than directly train on VOC, thus the comparison would be too complex.
